src/systems/personal-space-bubble.js

```diff
@@ -68,13 +68,7 @@ AFRAME.registerSystem("personal-space-bubble", {
   tick() {
     if (!this.data.enabled) return;
 
-    // Update matrix positions once for each space bubble and space invader
```

Comment:
I agree these calls to updateMatrixWorld seems unnecessary. You added them 7 months ago and if you don't remember having a specific reason for it, and the space bubble still works, I think it's fine to remove.

Reply:
I explained above that it's not that we don't remember why it isnt needed, it just happens to not be needed. It is certianly more "correct" to do the update, but it happens to be supurflous given the current configuration of things in our app. Luckily some work is being done in ThreeJS to make superflous updates like this free, so it will be correct to do the "correct" thing again.

Reply:
I added the comment you wished for.